airgun/views/computeresource.py

```diff
@@ -146,3 +146,5 @@ class virtual_machines(SatTab):
                     'Power': Text('.//span[contains(@class,"label")]'),
                 }
         )
+        vm_search = TextInput(
```

Comment:
Would `class virtual_machines(SatTab, SearchableViewMixin):` do the trick instead?

Reply:
I tried it but it didn't work.

Reply:
I'd try to extend our search field's locator then, since it's yet-another-search-field-with-completely-the-same-functionality-but-different-unique-attr 🙄 
I mean smth like:
```python
class Search(Widget):
    search_field = TextInput(locator=(
        ".//input[@id='search' or @type='search' or "
        "@ng-model='table.searchTerm' or contains(@ng-model, 'Filter')]"))
```
Could you please try it out?